client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-alert */
 import React, { useState } from 'react';
 import { Col, Row, Typography, Popconfirm, Tooltip, notification } from 'antd';
```

Comment:
Please remove notification because it is not used anymore
```suggestion
import { Col, Row, Typography, Popconfirm, Tooltip } from 'antd';
```